go/mysql/utils.go

```diff
@@ -203,3 +204,47 @@ func GetTableColumns(db *gosql.DB, databaseName, tableName string) (*sql.ColumnL
 	}
 	return sql.NewColumnList(columnNames), sql.NewColumnList(virtualColumnNames), nil
 }
+
+func versionTokens(version string, digits int) []int {
```

Comment:
@raidivyansh074 I suggest we use https://github.com/hashicorp/go-version for this instead so there is less code to maintain into the future